### PR TITLE
Update cocos2dx/platform/mac/CCGL.h

### PR DESCRIPTION
**Pull request by aninggo**
add VAO support

### CODE REVIEW

File: cocos2dx/platform/mac/CCGL.h [hunk 1]
```diff
@@ -30,4 +30,10 @@ of this software and associated documentation files (the "Software"), to deal
 
 #define CC_GL_DEPTH24_STENCIL8      -1
 
+
+#define glDeleteVertexArrays            glDeleteVertexArraysAPPLE
+#define glGenVertexArrays               glGenVertexArraysAPPLE
+#define glBindVertexArray               glBindVertexArrayAPPLE
+
+
```
Comment (minggo) [inline cocos2dx/platform/mac/CCGL.h:38]: Any benefit from this change?
Now mac port works well.

Comment (aninggo) [inline cocos2dx/platform/mac/CCGL.h:38]: enable CC_TEXTURE_ATLAS_USE_VAO  !  

#if CC_TEXTURE_ATLAS_USE_VAO
    //
    // Using VBO and VAO
    //
    glBindVertexArray( m_uVAOname );

Comment (minggo) [inline cocos2dx/platform/mac/CCGL.h:38]: So these codes should be changed too, right?
https://github.com/cocos2d/cocos2d-x/blob/gles20/cocos2dx/include/ccConfig.h#L175

Comment (aninggo) [inline cocos2dx/platform/mac/CCGL.h:38]: I think so.

Comment (minggo) [inline cocos2dx/platform/mac/CCGL.h:38]: So could you modify it too?
I think it is more suitable to modify it in this commit.

**Pull request merged**